libs/design/card/README.md

```diff
@@ -12,7 +12,9 @@ The example below is a default, filled card that includes all of a card's pre-st
 ## Usage
 
 ### Within a standalone component
-To use card in a standalone component, import it directly into your custom component:
+To use card in a standalone component, import it directly into your custom component.
+
+The following array exports can be used, depending on which card is being used: `DAFF_CARD_COMPONENTS`, `DAFF_RAISED_COMPONENTS`, `DAFF_STROKED_CARD_COMPONENTS`, and `DAFF_ALL_CARD_COMPONENTS`.
```

Comment:
I don't think we need to clarify that these are arrays, we can simply say:

The following imports are available, depending on which card your are choosing to use: `DAFF_CARD_COMPONENTS`, `DAFF_RAISED_COMPONENTS`, `DAFF_STROKED_CARD_COMPONENTS`. While we don't recommend it, if you need to use all the cards types available in your component, you can use `DAFF_ALL_CARD_COMPONENTS`.

Reply:
`While we don't recommend it` leaving this out because if you are using all card types within a single component, `DAFF_ALL_CARD_COMPONENTS` would be the correct import to use.

Reply:
updated